internal/clients/idler.go

```diff
@@ -92,6 +109,47 @@ func (i *idler) IsIdle(tenant string, openShiftAPIURL string) (bool, error) {
 	return s.IsIdle, nil
 }
 
+// Status returns true if the Jenkins instance for the specified tenant is idled, false otherwise.
+func (i *idler) Status(tenant string, openShiftAPIURL string) (string, error) {
+	namespace := tenant
+	if !strings.HasSuffix(tenant, namespaceSuffix) {
+		namespace = tenant + namespaceSuffix
+		log.WithField("ns", tenant).Debugf("Adding namespace suffix - resulting namespace: %s", namespace)
+	}
+	req, err := http.NewRequest("GET", fmt.Sprintf("%s/api/idler/status/%s", i.idlerAPI, namespace), nil)
+	if err != nil {
+		return "", err
+	}
+
+	q := req.URL.Query()
+	q.Add(OpenShiftAPIParam, util.EnsureSuffix(openShiftAPIURL, "/"))
+	req.URL.RawQuery = q.Encode()
+
+	log.WithFields(log.Fields{"request": logging.FormatHTTPRequestWithSeparator(req, " "), "type": "status"}).Debug("Calling Idler API")
+
+	client := &http.Client{}
+	resp, err := client.Do(req)
+	if err != nil {
+		return "", err
+	}
+	defer resp.Body.Close()
+
+	body, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		return "", err
+	}
+
+	s := &statusResponse{}
+	err = json.Unmarshal(body, s)
+	if err != nil {
+		return "", err
+	}
+
+	log.Debugf("Jenkins status is (%t) in %s", s.Data.State, namespace)
```

Comment:
Like in https://github.com/sthaha/fabric8-jenkins-proxy/commit/97d7357d13e5543d9878d6018c4e09c9a90ee060

It need to check if the Error is empty and return the first error